meta/documents/changelog_de.md

```diff
@@ -1,71 +1,75 @@
 # Release Notes für Lastschrift
 
-## 1.0.11 (2020-06-18)
+## 1.0.12
 
-### Gefixt
-- Die Prüfung ob die Zahlungsart für Gastbestellungen erlaubt ist wurde korrigiert.
+### Behoben
+- Der Link zu einer Internen Inforseite wird nun wieder korrekt angezeigt
```

Comment:
`internen Infoseite`. Also I think a `.` at the end is missing.

Reply:
yes, you're right. I changed this.